src/main/java/ch/njol/skript/expressions/ExprLastDamageCause.java

```diff
@@ -18,86 +18,97 @@
  */
 package ch.njol.skript.expressions;
 
+import org.bukkit.damage.DamageSource;
+import org.bukkit.damage.DamageType;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
 import ch.njol.skript.bukkitutil.HealthUtils;
 import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.PropertyExpression;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.util.Getter;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
-import org.bukkit.entity.LivingEntity;
-import org.bukkit.event.Event;
-import org.bukkit.event.entity.EntityDamageEvent;
-import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
-import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author bensku
- *
- */
-@Name("Last Damage Cause")
-@Description("Cause of last damage done to an entity")
-@Examples({"set last damage cause of event-entity to fire tick"})
+@Name("Last Damage Cause/Type")
+@Description({
+	"Cause of last damage done to an entity.",
+	"Using 'damage type' is more accurate as it includes data pack types, but it is only available on versions 1.20.4 and newer."
+})
+@Examples({
+	"set last damage cause of event-entity to fire tick"
```

Comment:
Since last damage cause is now deprecated this example should change and if there is only 1 example we don't need the list braces